core/src/main/java/de/muenchen/allg/itd51/wollmux/former/BroadcastObjectSelection.java

```diff
@@ -23,8 +23,8 @@
 package de.muenchen.allg.itd51.wollmux.former;
 
 /**
- * Nachricht, dass in einer View ein Object ausgewählt wurde. Diese Nachricht wird
- * von anderen Views ausgewertet, um ihre Selektionen ebenfalls anzupassen.
+ * Message that an object was selected in a view. This message will be
```

Comment:
bit bumpy - how about: ..message for notifying object selection in a View.. (please upper-case View, it's a noun & used as classname elsewhere)